### ISSUE TITLE
Bot chat not triggering webhook, Postman direct POST does

### ISSUE THREAD

**Issue by CapitanFindusFI** [php-telegram-bot/core#577]
### Required Information
- Operating system: Windows 10
- PHP version: 5.6
- PHP Telegram Bot version: 0.46
- Using MySQL database: yes 
- MySQL version: 5.5.43
- Update Method: Webhook 
- Self-signed certificate: no

### Expected behaviour
Bot auto-answering

### Actual behaviour
Bot is not answering from chat, only answers by using Postman POST to my hook

### Steps to reproduce
Created a webhook, webhook is working on HTTPS, getWebhookInfo is correct.

BotController.php constructor and Hook handler
```
function __construct() {
	$this->telegram = new Telegram(getenv('TELEGRAM_BOT_APIKEY'), getenv('TELEGRAM_BOT_NAME'));
	$this->bot_username = getenv('TELEGRAM_BOT_NAME');

	TelegramLog::initErrorLog(__DIR__ . '/' . $this->bot_username . '_error.log');
	TelegramLog::initDebugLog(__DIR__ . '/' . $this->bot_username . '_debug.log');
	TelegramLog::initUpdateLog(__DIR__ . '/' . $this->bot_username . '_update.log');

	$this->telegram->enableMySQL([
		"host" => getenv('DB_HOST'),
		"user" => getenv('DB_USER'),
		"database" => getenv('DB_NAME'),
		"password" => getenv('DB_PASS')
	]);

	$this->telegram->enableAdmin((int)getenv('TELEGRAM_ADMIN_ID'));

	$this->telegram->addCommandsPath(realpath(__DIR__ . '/..') . '/Commands');
}

public function handleWebhook(ServerRequestInterface $request, ResponseInterface $response) {
	try {
		$this->telegram->handle();
	} catch (TelegramException $e) {
		TelegramLog::error($e->getMessage());
	}
	return $response;
}
```

CiteCommand.php

```
class CiteCommand extends Command
{
    /**
     * @var string
     */
    protected $name = 'cite';
    /**
     * @var string
     */
    protected $description = 'Random Boris cite';
    /**
     * @var string
     */
    protected $usage = '/cite';
    /**
     * @var string
     */
    protected $version = '0.3.0';
    /**
     * @var bool
     */
    protected $need_mysql = true;
    /**
     * Conversation Object
     *
     * @var \Longman\TelegramBot\Conversation
     */
    protected $conversation;

    /**
     * Command execute method
     *
     * @return \Longman\TelegramBot\Entities\ServerResponse
     * @throws \Longman\TelegramBot\Exception\TelegramException
     */
    public function execute() {
        $message = $this->getMessage();
        $chat = $message->getChat();
        $chat_id = $chat->getId();
        //Preparing Response
        $data_controller = new DataController();
        $data = [
            'chat_id' => $chat_id,
            'text' => $data_controller->randomCite()
        ];
        $result = Request::sendMessage($data);
        return $result;
    }
}
```

### Extra details
I'm using League\Route to handle PHP routes, which needs controllers to use ServerRequestInterface and ResponseInterface as default callable parameters

**Comment by jacklul** [comment]
Is the message from chat request inserted into database? If it's not then Telegram can't reach webhook due to misconfiguration on your side.

**Comment by CapitanFindusFI** [comment]
@jacklul thank you for answering

When I answer bot by using Telegram chat, DB doesn't update. It does by triggering Postman POST directly.
Could it be a cURL block? I can't see errors anywhere, neither on error logs or access log

**Comment by jacklul** [comment]
Maybe certificate issue, try `https://api.telegram.org/botYOURTOOKEN/WebhookInfo`, look at the webhook error field, it might tell you what's wrong!

**Comment by CapitanFindusFI** [comment]
This is what I get on getWebhookInfo

```
{
	"ok": true,
	"result": {
		"url": "https://myurl/telegram-bot/api/hooks/",
		"has_custom_certificate": false,
		"pending_update_count": 0,
		"max_connections": 40
	}
}
```

**Comment by jacklul** [comment]
There is no error there so no, there isn't. 
There must be server side issue with routing into webhook, I would suggest debugging your code in few places to see how far execution goes after messaging your bot.

**Comment by noplanman** [comment]
Does the update log get written?

Another easy way to debug, is to use a raw update JSON string and execute with getUpdates method from CLI. That way, any PHP errors or warnings get output.

Also, your command shouldn't be extending `Command`, but instead either `UserCommand` or `AdminCommand`.

**Comment by CapitanFindusFI** [comment]
By contacting hosting provider I've found out that server is responding only to my IP address,
cURLs are blocked from outside without reason. After unblocking cURL, error logs started becoming useful, so I've found another problem in my route handling (because of trailing slash).

Solving these problems, and bot finally started working! This is awesome!

@noplanman Thank you for the tip, I wrote a `Command` just for testing, I had no clue on how to solve my problem.
